README.md

```diff
@@ -1,7 +1,19 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
 
+# The databases can be found using the local databases as follows:
+PGDATABASE=nc_news;
+PGDATABASE=nc_news_test;
+
+# The repository can be cloned with the following URL:
+
+https://github.com/lucMarteau/BE-NC-News.git
+
+# The following dependancies will be needed to run the files:
+
+npm install pg
+npm install jest
+...
```

Comment:
Just `npm install` would automatically install all the deps listed in the package.json, so anyone cloning your wouldn't have to install them one by one. (even though you do for the first install)